src/utils/collapseSourcemaps.ts

```diff
@@ -174,34 +168,55 @@ export default function collapseSourcemaps(
 			},
 			[source]
 		);
+	};
+}
+
+function unionSourcemaps(
+	id: string,
+	originalCode: string,
+	originalSourcemap: ExistingDecodedSourceMap | null,
+	sourcemapChain: DecodedSourceMapOrMissing[],
+	linkMap: (source: Source | Link, map: DecodedSourceMapOrMissing) => Link
+) {
```

Comment:
Not strictly necessary but I would find it helpful for understanding the code if the inferred return value type could be added explicitly to the function declaration?